src/components/nav-icon-item/styles.ts

```diff
@@ -0,0 +1,52 @@
+import { AnchorHTMLAttributes, ButtonHTMLAttributes } from 'react'
+import { styled } from '@linaria/react'
+import { ElIcon } from '../icon'
+
+const baseStyles = `
+  display: inline-flex;
+  padding: var(--space-2, 8px);
+  justify-content: center;
+  align-items: center;
+  gap: var(--space-none, 0px);
+  border-radius: var(--corner-default, 4px);
+  background: var(--fill-white, #ffffff);
+  border: var(--border-none, 0px);
+  color: var(--icon-app_bar-default, #798da1);
+  outline: none;
+
+  &:focus-visible {
+    border-radius: var(--corner-default, 4px);
```

Comment:
do we not have a global focus style?

Reply:
> do we need a global focus style?

**thought:** it could be useful, but I'm unsure if it's wise, as nearly all AppBar items share the same global focus style

![{B117EE35-67E4-4AD2-9FFD-2AF91D1AA1CA}](https://github.com/user-attachments/assets/5113af73-2f82-4668-88c9-f430c661e0e3)

Reply:
Yeah, we should really have a design token exported from Figma for the focus style's box shadow, but I believe all the design tokens require a refactor, so I'm not sure it's worth getting Design to specify one now or not 🤷‍♂️ Probably just need to ask them what they think...